appeals/web/src/server/appeals/appeal-details/manage-linked-appeals/__tests__/__snapshots__/manage-linked-appeals.test.js.snap

```diff
@@ -987,6 +892,94 @@ exports[`linked-appeals GET /linked-appeals/add/check-and-confirm should render
 </main>"
 `;
 
+exports[`linked-appeals GET /linked-appeals/manage should render the manage linked appeals page with the expected content when the appeal is a child 1`] = `
+"<main class="govuk-main-wrapper " id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full"><span class="govuk-caption-l govuk-!-margin-bottom-3">Appeal 351062</span>
+            <h1             class="govuk-heading-xl govuk-!-margin-bottom-3">Manage linked appeals</h1>
+        </div>
+    </div>
+    <div class="govuk-!-margin-bottom-2"><strong class="govuk-tag govuk-tag--grey single-line">Child</strong>
+    </div>
+    <h2 class="govuk-!-margin-top-6">Lead appeal of 351062</h2>
+    <table class="govuk-table">
+        <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+                <th scope="col" class="govuk-table__header">Appeal ID</th>
+                <th scope="col" class="govuk-table__header">Appeal type</th>
+                <th scope="col" class="govuk-table__header">Action</th>
+            </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+            <tr class="govuk-table__row">
+                <td class="govuk-table__cell"><a class="govuk-link" href="/appeals-service/appeal-details/3" aria-label="Appeal 7 2 5 2 8 4">725284</a>
+                </td>
+                <td class="govuk-table__cell">Householder</td>
+                <td class="govuk-table__cell"><a class="govuk-link" href="/appeals-service/appeal-details/1/linked-appeals/unlink-appeal/2/3048/2">Unlink</a>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+    <h2 class="govuk-!-margin-top-6">Other child appeals of 351062</h2>
+    <table class="govuk-table">
+        <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+                <th scope="col" class="govuk-table__header">Appeal ID</th>
+                <th scope="col" class="govuk-table__header">Appeal type</th>
+                <th scope="col" class="govuk-table__header">Action</th>
+            </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+            <tr class="govuk-table__row">
+                <td class="govuk-table__cell"><a class="govuk-link" href="/appeals-service/appeal-details/4" aria-label="Appeal 7 6 2 1 5 4 1 6">76215416</a>
+                </td>
+                <td class="govuk-table__cell">Unknown</td>
```

Comment:
Is this unknown because it's a Horizon appeal? Should we pull the appeal type through?

Reply:
Ah nevermind, I see it's a value from the test (managed-linked-appeals.test.js ln39)

Reply:
BTW if need to store the horizon appeal type, I've added a `externaAppealType` field to the `AppealRelationship` table...

Reply:
I think it was this way already before my fix - we could add that but I think it should go through design first and be part of a separate ticket